static/usr/lib/systemd/system/plymouth-quit.service.d/core-override.conf

```diff
@@ -0,0 +1,6 @@
+[Unit]
+After=snapd.seeded.service
+ConditionKernelCommandLine=snapd_recovery_mode=run
+
+[Service]
+ExecStartPost=-systemctl stop splash-client.service
```

Comment:
I would use `plymouth --ping` in the loop of `splash-client` instead so we do not have to change `plymouth-quit.service`.